Dockerfile

```diff
@@ -0,0 +1,219 @@
+	composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --classmap-authoritative; \
+	composer clear-cache
+ENV PATH="${PATH}:/root/.composer/vendor/bin"
+
+WORKDIR /srv/sylius
+
+RUN git clone --depth 1 --single-branch --branch "1.7" https://github.com/Sylius/Sylius-Standard.git .
+
+RUN set -eux; \
+	cp docker/php/php.ini /usr/local/etc/php/php.ini; \
+	cp docker/php/php-cli.ini /usr/local/etc/php/php-cli.ini
+
+COPY . themes/BootstrapTheme
+
+# build for production
+ARG APP_ENV=prod
+
+RUN set -eux; \
+	sed -i s/2G/6G/g /usr/local/etc/php/php-cli.ini; \
+	composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
+	composer require symfony/webpack-encore-bundle; \
+	composer clear-cache
+
+COPY docker/assets.yaml config/packages/assets.yaml
+COPY docker/webpack_encore.yaml config/packages/webpack_encore.yaml
+
+COPY docker/SyliusAdminBundle templates/bundles/SyliusAdminBundle
+COPY docker/SyliusShopBundle templates/bundles/SyliusShopBundle
+
+
+FROM node:${NODE_VERSION}-alpine AS sylius_nodejs
+
+WORKDIR /srv/sylius
+
+RUN set -eux; \
+	apk add --no-cache --virtual .build-deps \
+		g++ \
+		gcc \
+		git \
+		make \
+		python \
+	;
+
+# prevent the reinstallation of vendors at every changes in the source code
+COPY --from=sylius_php_base /srv/sylius/package.json ./
+COPY --from=sylius_php_base /srv/sylius/yarn.lock ./
+
+RUN set -eux; \
+	yarn add \
+		@symfony/webpack-encore \
+		sass-loader@^8.0.0 \
+		node-sass \
+		lodash.throttle -D \
+	;\
+	yarn add \
+		bootstrap@^4.5.0 \
+		bootstrap.native@^3.0.0 \
+		glightbox \
+		axios \
+		form-serialize \
+		@fortawesome/fontawesome-svg-core \
+		@fortawesome/free-brands-svg-icons \
+		@fortawesome/free-regular-svg-icons \
+		@fortawesome/free-solid-svg-icons \
+	; \
+	yarn install; \
+	yarn cache clean
+
+COPY . themes/BootstrapTheme
+COPY --from=sylius_php_base /srv/sylius/vendor/sylius/sylius/src/Sylius/Bundle/UiBundle/Resources/private vendor/sylius/sylius/src/Sylius/Bundle/UiBundle/Resources/private/
+COPY --from=sylius_php_base /srv/sylius/vendor/sylius/sylius/src/Sylius/Bundle/AdminBundle/Resources/private vendor/sylius/sylius/src/Sylius/Bundle/AdminBundle/Resources/private/
+COPY --from=sylius_php_base /srv/sylius/vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/private vendor/sylius/sylius/src/Sylius/Bundle/ShopBundle/Resources/private/
+COPY --from=sylius_php_base /srv/sylius/assets assets/
+COPY --from=sylius_php_base /srv/sylius/webpack.config.js ./webpack.config.js.orig
+
+COPY docker/webpack.config.js.part ./
+RUN set -eux; \
+	head -n -1 webpack.config.js.orig > webpack.config.js; \
+	cat ./webpack.config.js.part >> webpack.config.js
+
+RUN yarn encore production
+
+COPY --from=sylius_php_base /srv/sylius/docker/nodejs/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
+RUN chmod +x /usr/local/bin/docker-entrypoint
+
+ENTRYPOINT ["docker-entrypoint"]
+CMD ["yarn", "encore", "dev", "--watch"]
+
+
```

Comment:
Remove this line

Reply:
The extra line makes it easier to see that the separation between between the stages is here. Shall I replace it with a comment block to clarify that a new stage begins here?

Reply:
Comment block could be nice (something like this: https://github.com/Sylius/Sylius-Standard/pull/391/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R132)